### PR TITLE
WIP Express 2

### PR DESCRIPTION
**Pull request by jshawl**

### CODE REVIEW

**Comment by jshawl** [comment]
cc @adambray

**Comment by mattscilipoti** [comment]
Surprised to see RegEx removed.  Where are we moving it too?  We just keep waving hands at RegEx and pushing it down the road.  I was expecting to see an improved exercise, rather than removal.

**Comment by jshawl** [comment]
felt out of place here. objectives for express 2 lesson:

Prepare students to build an MVC organized express application that handles both Html and JSON requests.

**Comment by mattscilipoti** [comment]
re: app.head.  It's an often overlooked, little used method that developers should be taking advantage of.  This felt like a good time to briefly discuss head and share that there is a much bigger world (26 request methods).

**Comment by adambray** [comment]
I'm in favor of removing regex here (offering an optional 1 hr lesson one afternoon next week). As for HEAD, I'm on the fence, but leaning slightly in favor of keeping for the reasons Matt mentioned.

File: 08-single-page-web-apps/express_2/readme.md [hunk 1]
```diff
@@ -0,0 +1,210 @@
...
+  {{#each compliments}}
+    <li>{{this}}</li>
+  {{/each}}
+</ul>
+```
+
+> Bonus: show a random color when the page loads
+
+## I Do: Forms, bodyParser, and creating compliments
+
```
Comment (andrewsunglaekim) [inline 08-single-page-web-apps/express_2/readme.md]: Possible to have more explicit code here?

File: 08-single-page-web-apps/express_2/readme.md [hunk 2]
```diff
@@ -0,0 +1,210 @@
...
+``` js
+// root route
+app.get('/', function (req, res){
+  res.redirect('/songs');
+});
+```
+
+## You do: edit compliments
+
```
Comment (andrewsunglaekim) [inline 08-single-page-web-apps/express_2/readme.md]: needs more explicit directions

File: 08-single-page-web-apps/express_2/readme.md [hunk 3]
```diff
@@ -0,0 +1,210 @@
...
+- Really like the organization I see in [this tutorial](https://thewayofcode.wordpress.com/2013/04/21/how-to-build-and-test-rest-api-with-nodejs-express-mocha/)
+- Looking for more?  We recommend NodeSchool's [ExpressWorks](https://github.com/azat-co/expressworks)
+
+
+### Express Route Tester
+
+Similarly, let's take a look at the [Express Route Tester](http://forbeslindesay.github.io/express-route-tester/?_ga=1.256234632.2070291842.1433362238).  It can be a handy tool for testing basic Express routes.
+
+
+## Conclusion
```
Comment (andrewsunglaekim) [inline 08-single-page-web-apps/express_2/readme.md]: also maybe cliffnotes

**Pull request merged**